### ISSUE TITLE
bring back the dark/light ages...

### ISSUE THREAD

**Issue by nathanjhood** [StoneyDSP/stoneydsp.com#187]
# Chore

## Describe the chore

Finish porting the dark mode toggle Js, as currently found the 'app' project - probably needs to be Typescript, *and* pass the linting config.

Add the cookie for dark/light mode preference to the user db???

**Comment by nathanjhood** [comment]
[Tailwind docs](https://tailwindcss.com/docs/dark-mode)

**Comment by nathanjhood** [comment]
> Add the cookie for dark/light mode preference to the user db???

Should be possible now since implementing a global cookie store on #249

**Comment by nathanjhood** [comment]
Finally made some improvements here, on #357 

So far still just using system pref, but the actual tailwind/CSS config is more clear (to me, now) and much easier to work with.

**Comment by nathanjhood** [comment]
Current status:

Light/dark mode CSS is implemented at system-preference level. NO cookies are used. Also, I added a default call to the dark mode for when the user has no system preference.

This all ties in kind of nicely with for example Vercel Analytics, which also uses no cookies.

So in it's current form, without a manual light/dark mode cookie, we've covered the functionality without needing to store anything on either local or session storage. 

That means we can make our Terms of Service quite a bit more 'user-friendly' due to the un-obtrusive nature of the site. (side note: a toast notification would be worth having in either case)

Going in the other direction, I have been pondering using the github-readme-stats themes (which I've bundled) as a site-wide application with user control, just to be showy - it's a portfolio, after all.

This ticket is quite outdated now and there are deeper considerations and design decisions at play.

Will likely revisit in separated aspects under new tickets, when the time comes.
